src/WCSimTrackingAction.cc

```diff
@@ -18,6 +18,9 @@ WCSimTrackingAction::WCSimTrackingAction()
 
   ProcessList.insert("Decay") ;                         // Michel e- from pi+ and mu+
   ProcessList.insert("conv") ;                         // Products of gamma conversion
+  ProcessList.insert("phot");
+  ProcessList.insert("comp");
+  ProcessList.insert("NeutronInelastic");
```

Comment:
Can you add comments to these lines, to explain in more detail what they are? (can be short descriptions as above)

Reply:
I added this some time ago because I thought I wasn't seeing the particles created by those processes in my DataFrames because they were not in the ProcessList. Anyway, I just commented those lines and I see them, so everything is fine.